### ISSUE TITLE
[Teaser] Loosing Styles After Alt Text Error

### ISSUE THREAD

**Issue by juliaetate** [adobe/aem-core-wcm-components#2943]
## Bug Report

**Current Behavior**
In the Teaser component, if you have an Asset with no alt text stored and "Don't provide an alternative text" selected with a Styles option selected, and you edit the component to uncheck the "Don't provide an alternative text" checkbox without filling in the alt text field, it will produce an error. When the alt text field has an error, the Styles options are disabled. If you then check the box to "Inherit from description of asset", you are allowed to save the component but the Alt text error doesn't fully resolve. You can still see the error text and the Styles options remain disabled although you are able to save the component. The alt text field is still populated and will save correctly and the alt text will be visible in the HTML, however, the Styles tab options will be cleared.

**Expected behavior/code**
If the "Inherit from description of asset" is selected without interacting with the alt text field, the alt test error should still fully resolve AND the Styles tab fields should be enabled to prevent losing the Styles tab selections. 

**Environment**
- AEM SDK Version - 2025.5.21005.20250522T173058Z-250400
- Core Components version 2.29.0
- JRE version 11.0.27 2025-04-15 LTS

**Possible Solution**
We have been able to overlay the JS for the Image component which also drives this part of the Teaser and other components built from it to clear the error message but we were not able re-enable the Styles tab fields. When we tried to re-enable those fields, they would go back to disabled once we tried to interact with the Styles tab. The Styles tab fields will be re-enabled if an author types directly into the alt text field. 

We have also experienced the issue in older AEM and Core Components version.

**Additional context / Screenshots**
Here is a screen recording of the steps and issue.
